kb/deadline-exceeded-troubleshooting.md

```diff
@@ -0,0 +1,99 @@
+---
+slug: deadline-exceeded-troubleshooting
+title: DeadlineExceeded error troubleshooting
+tags:
+  - kb-article
+date: 2022-11-08T00:00:00Z
+---
+
+All requests made to the [Temporal Cluster](/concepts/what-is-a-temporal-cluster) are [gRPC requests](https://grpc.io/docs/what-is-grpc/core-concepts/#deadlines).
```

Comment:
Maybe could clarify where these requests originate from, namely users client code and their workers.
Also if we mention this we could say that all these calls are described via apis: https://github.com/temporalio/api/tree/master/temporal/api